CONTRIBUTING.rst

```diff
@@ -59,7 +59,12 @@ to see how that works.
 
 Updating requirements
 ---------------------
-To update requirements edit ``requirements/requirements.in``
+To update requirements edit
+``requirements/requirements.in`` for packages for all environments
+``requirements/production.in`` for packages for production environment only
+``requirements/tests.in`` for packages for test environment only
+``requirements/dev.in`` for packages for dev environment only
```

Comment:
I think besides `requirements.in` these filenames are wrong, but otherwise +1 on the change

Reply:
sorry about that

(note to self: don't try to make a PR at 1% battery)